lib/core/src/lib/datatable/components/datatable-cell/datatable-cell.component.ts

```diff
@@ -41,12 +41,14 @@ import { takeUntil } from 'rxjs/operators';
                 [attr.aria-label]="value$ | async"
                 [title]="tooltip"
                 class="adf-datatable-cell-value"
+                role="button"
```

Comment:
Do we want to set button role to each table cell? It is quite multipurpose component in general. @DenysVuika what do you think?

Reply:
hmm, cells are not buttons, correct

Reply:
Well, that is interesting point. I was not sure as well so thank you for raising that here. I needed to add button role because of accessibility request from CSV file:
<img width="1352" alt="image" src="https://user-images.githubusercontent.com/115619721/200588254-912d67be-a823-43f6-8e86-fecc467b4c69.png">
https://axeauditor.dequecloud.com/api/v1/file/91e90b78-f1f3-11ec-b66c-0748e7218481

You see that mentions that we need to add that role to these elements. 
Do you have any suggestion how should I fix it instead?

Reply:
Honestly nothing good comes to my mind right now. @DenysVuika any ideas for that?

Reply:
Sometimes a11y tools suggest completely stupid solutions, so we should be still evaluating scenarios with the common sense. Table cells can never be buttons

Reply:
I think the reason why it could think it is button by mistake is that you can focus cell using tab and "click" on that cell using enter so then you enter into folder: 
<img width="669" alt="image" src="https://user-images.githubusercontent.com/115619721/200769983-9d4b6c28-fa8b-47cd-864b-8a9673a7abac.png">

But sure we can skip that point. I will remove these roles in next commit in that PR

Reply:
Done